training/finetune_acft_hf_nlp/src/model_selector/model_selector.py

```diff
@@ -82,6 +82,15 @@ def model_selector(args: Namespace):
     task_runner = get_task_runner(task_name=args.task_name)()
     task_runner.run_modelselector(**vars(args))
 
+    # for base curated models forward MLmodel info
+    if getattr(args, "mlflow_model_path", None) is not None:
+        import shutil
+        from azureml.acft.contrib.hf.nlp.constants.constants import MLFlowHFFlavourConstants
+        mlflow_config_file = Path(args.mlflow_model_path, MLFlowHFFlavourConstants.MISC_CONFIG_FILE)
+        if mlflow_config_file.is_file():
+            shutil.copy(str(mlflow_config_file), args.output_dir)
+            logger.info("Copied MLmodel file to output dir")
```

Comment:
log a message if the file does not exist

Reply:
done